mountpoint-s3/tests/fuse_tests/write_test.rs

```diff
@@ -731,7 +731,7 @@ where
 
     // Open with O_RDWR and write something to the file
     let mut options = File::options();
-    let mut write_fh = options.read(true).write(true).open(&path).unwrap();
+    let mut write_fh = options.read(true).write(true).truncate(true).open(&path).unwrap();
```

Comment:
Do we have tests for the non-truncate case (which should fail)? And a test for opened with O_TRUNC but closed without writing anything?

Reply:
Added them in the new commit